### PR TITLE
samples: nrf9160: gps: simplify flag check; use k_msleep

### PR DESCRIPTION
**Pull request by amarkevich**
Signed-off-by: Alexey Markevich <buhhunyx@gmail.com>

### CODE REVIEW

File: samples/nrf9160/gps/src/main.c [hunk 1]
```diff
@@ -309,9 +309,7 @@ int process_gps_data(nrf_gnss_data_frame_t *gps_data)
 			nmea_string_cnt = 0;
 			got_fix = false;
 
-			if ((gps_data->pvt.flags &
-				NRF_GNSS_PVT_FLAG_FIX_VALID_BIT)
-				== NRF_GNSS_PVT_FLAG_FIX_VALID_BIT) {
+			if (gps_data->pvt.flags & NRF_GNSS_PVT_FLAG_FIX_VALID_BIT) {
```
Comment (rlubos) [inline samples/nrf9160/gps/src/main.c:312]: This line exceeds 80 char/line limit now.

File: samples/nrf9160/gps/src/main.c [hunk 2]
```diff
@@ -446,7 +444,7 @@ int main(void)
 			printk("---------------------------------");
 		}
 
-		k_sleep(K_MSEC(500));
+		k_msleep(500);
```
Comment (rlubos) [inline samples/nrf9160/gps/src/main.c:447]: I'd be consistent and switch to `k_msleep` in other places as well (line 344).

**Comment by rlubos** [comment]
@evenl Please check if you're ok with the proposed changes.

**Review by evenl — APPROVED** [review]

**Review by lemrey — APPROVED** [review]
LGTM

**Review by rlubos — APPROVED** [review]

**Pull request merged**